java-tools/OsmAndServer/src/main/java/net/osmand/server/api/searchtest/entity/Dataset.java

```diff
@@ -0,0 +1,94 @@
+package net.osmand.server.api.searchtest.entity;
```

Comment:
Files are too small all package could fit several files, as we store in the project. Move it to repository Dataset -> DatasetRepository...